Shopify/cli
# Improved language around --nodelete flag in push and pull commands

### WHY are these changes introduced?
Create some clarity around wording for `--nodelete` flag

References: https://github.com/Shopify/develop-advanced-edits/issues/225

### WHAT is this pull request doing?
Updating the language for the `--nodelete` flag on push and pull commands

### How to test your changes?

Purely documentation changes but you can see them by 
- pulling down this branch
- building the branch
     Running:
  - `shopify theme push --help`
  - `shopify theme pull --help`

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
